pkg/action/install.go

```diff
@@ -421,7 +424,7 @@ func (i *Install) replaceRelease(rel *release.Release) error {
 }
 
 // renderResources renders the templates in a chart
-func (c *Configuration) renderResources(ch *chart.Chart, values chartutil.Values, outputDir string, subNotes bool) ([]*release.Hook, *bytes.Buffer, string, error) {
+func (c *Configuration) renderResources(ch *chart.Chart, values chartutil.Values, releaseName string, outputDir string, subNotes bool, useReleaseName bool) ([]*release.Hook, *bytes.Buffer, string, error) {
```

Comment:
Nit: consider short handing the last 2 args to `subNotes, useReleaseName bool`

Reply:
Updated